src/config/example.toml

```diff
@@ -40,27 +37,21 @@ async_stack_trace = "On"
 unique_user_stream_errors = 10
 
 [streaming.developer]
-batch_output_channel_size = 64
-batch_chunk_size = 1024
 stream_enable_executor_row_count = false
```

Comment:
Do we need to maintain backward compatibility for developer configs? If not, I guess we can now remove the prefix `stream_`. cc @fuyufjh

Reply:
It's fine I think, there was disclaimer about these configs in developer section before, i.e. we don't provide any guarantee